src/operations.rs

```diff
@@ -575,8 +573,8 @@ fn _remove_symlink(symlink_path: &Path) -> Result<()> {
     Ok(())
 }
 
-pub fn remove_symlink(symlink_name: &String) -> Result<()> {
-    let symlink_path = get_bin_dir()
+pub fn remove_symlink(symlink_name: &String, paths: &GlobalPaths) -> Result<()> {
+    let symlink_path = get_bin_dir(paths)
```

Comment:
See comment (second half): https://github.com/JuliaLang/juliaup/pull/886#issuecomment-2086510625
In short: I don't get why the `get_bin_dir` uses an undocumented environment variable `JULIAUP_BIN_DIR`.
If we want to depend on `current_exe()` then that makes `JULIAUP_BIN_DIR` a potential source of headaches, since almost by definition, the path obtained by `current_exe()` *is* the actual `JULIAUP_BIN_DIR` (unless I am not seeing the use case for it)

If the `JULIAUP_BIN_DIR` was exclusively introduced (in #165 ) as a kitchen sink for various versions' symlinks then I guess it's not such a big deal. But I must admit I still find it confusing, especially since it is not documented and so it seems like a left over from a past approach?